central-login/package.json

```diff
@@ -20,11 +20,11 @@
     "webpack-dev-server": "^4.15.1"
   },
   "dependencies": {
-    "@forgerock/javascript-sdk": "^4.2.0"
+    "@forgerock/javascript-sdk": "^4.4.0"
   },
   "scripts": {
     "build": "webpack",
-    "start": "webpack serve --open",
+    "start": "webpack serve",
```

Comment:
I found it super annoying to have it reopen the window, especially as you need to be in a different browser than the one you're logged into ForgeRock/PingOne, so I removed it from this sample, as well as the embedded-login sample.